source/documentation/troubleshooting/ssh.md

```diff
@@ -64,23 +64,100 @@ buildpack: staticfile_buildpack
 
 There are 3 instances, with instance indexes from 0 to 2.
 
-If you have multiple instances like this and use `cf-ssh`, you will be connected to the app with an instance index of 0. 
+If you have multiple instances like this and use `cf-ssh`, you will be connected to the app with an instance index of 0.
 
 You can connect to a particular instance. For example, if you want to connect to instance 2, you can do this:
 
 ```
 cf ssh --app-instance-index 2
 ```
 
+## Creating TCP tunnels with SSH
 
+The `cf ssh` command supports [local port forwarding](https://en.wikipedia.org/wiki/Port_forwarding#Local_port_forwarding), which allows you to create tunnels from your local system to the application instance container. This is useful when you want to connect from your local system to a backing service that is only accessible from an app running on GOV.UK PaaS.
+
+To enable local port forwarding, you can use the parameter `-L`:
+
+```
+cf ssh APPNAME -L LOCALPORT:REMOTEHOST:REMOTEPORT
+```
+
+This will forward the `LOCALPORT` port on the local system to the given `REMOTEHOST` host and `REMOTEPORT` port on the application container side.  
+
+Whenever a connection is made to this port, the connection is forwarded over the secure SSH channel, and a connection is made to the host and port `REMOTEHOST:REMOTEPORT` from the remote application container.
+
+You can use the `-L` parameter multiple times to forward different ports.
+
+The tunnel will be closed once the `cf ssh` command is stopped.
+
+For example, you can connect directly to the PostgreSQL service bound to an application following these steps:
+
+ 1. Find the details of the service using `cf env APPNAME`. Here is some simplified and shortened example output:
+
+    ```
+    $ cf env myapp
+    Getting env variables for app myapp in org myoth / space myorg as randomuser...
+    OK
+
+        System-Provided:
+        {
+        "VCAP_SERVICES": {
+         "postgres": [
+         {
+         "credentials": {
+         "host": "rdsbroker-01-ff-d2.cwm.eu-west-1.rds.amazonaws.com",
+         "jdbcUrl": "jdbc:postgresql://rdsbroker-01-ff-d2.cwm.eu-west-1.rds.amazonaws.com:5432/rdsbroker_9f0_97_aa4?user=rdsbroker_9f0_97_aa4_owner\u0026password=xnYXthsgUFwPUOO",
+         "name": "rdsbroker_9f0_97_aa4",
+         "password": "xnYXthsgUFwPUOO",
+         "port": 5432,
+         "uri": "postgres://rdsbroker_9f0_97_aa4_owner:xnYXthsgUFwPUOO@rdsbroker-01-ff-d2.cwm.eu-west-1.rds.amazonaws.com:5432/rdsbroker_9f0_97_aa4",
+         "username": "rdsbroker_9f0_97_aa4_owner"
+        },
+        ...
+    ```
+
+    You will need to know:
+
+    + the remote host, displayed as `"host":`
+    + the remote port, displayed as `"port"`.
+    + the PostgreSQL username, displayed as `"username":`
+    + the PostgreSQL password, displayed as `password:`
+    + the name of the database, displayed as `name:`
```

Comment:
Should these be an unordered list?

At the moment they all render within a codeblock: https://docs.cloud.service.gov.uk/#creating-tcp-tunnels-with-ssh